### PR TITLE
plat-stm: main.c now use generic_boot.c

### PR DESCRIPTION
**Pull request by pascal-brand-st-dev**
Yet another step towrds generic boot on plat-stm
plat-stm/main.c is now minimized by including
kernel/generic_boot.c

Signed-off-by: Pascal Brand <pascal.brand@st.com>

### CODE REVIEW

File: core/arch/arm/plat-stm/main.c [hunk 1]
```diff
@@ -24,53 +24,31 @@
...
-extern uint8_t __heap2_end[];
-extern uint8_t __pageable_part_start[];
-extern uint8_t __pageable_part_end[];
-extern uint8_t __pageable_start[];
-extern uint8_t __pageable_end[];
+/*
+ * Going to generic boot
+ * Temporary step
+ */
+#include "../kernel/generic_boot.c"
```
Comment (jbech-linaro) [inline core/arch/arm/plat-stm/main.c]: Can't we workaround this already now?

**Comment by jbech-linaro** [comment]
Even though I have a comment about the included c-file  ...
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

**Comment by pascal-brand-st-dev** [comment]
`CFG_GENERIC_BOOT` makes use of `generic_boot.c`, `generic_core_bootcfg.c` and `generic_entry_a32.S`. We still miss the use of `generic_entry_a32.S` on plat-stm right now.

**Comment by jenswi-linaro** [comment]
```
Reviewed-by: Jens Wiklander <jens.wiklander@linaro.org>
```

**Pull request merged**